sdk/eventhub/azure-eventhub-checkpointstoreblob-aio/README.md

```diff
@@ -97,6 +97,20 @@ if __name__ == '__main__':
     loop.run_until_complete(main())
 ```
 
+#### Use `BlobCheckpointStore` with a different version of Azure Storage Service API
+Some environments have different versions of Azure Storage Service API. For instance, some Azure Stack use 2017-11-09.  
```

Comment:
"some Azure Stack deployments"?

Reply:
Not sure if we have other deployments that are not Azure Stack

Reply:
In that case I would change it to "For instance, Azure Stack may use...."

Reply:
Changed